SimTKcommon/Geometry/src/PolygonalMesh.cpp

```diff
@@ -321,8 +321,8 @@ void PolygonalMesh::loadVtpFile(const String& pathname) {
     Xml::Element points   = piece.getRequiredElement("Points");
     const int numPoints = 
         piece.getRequiredAttributeValueAs<int>("NumberOfPoints");
-    const int numPolys  = 
-        piece.getRequiredAttributeValueAs<int>("NumberOfPolys");
+    const unsigned int numPolys  = 
```

Comment:
Why were all these ints made unsigned? Where was the warning and why couldn't it have been fixed just with a cast somewhere? Making things unsigned is dangerous because simple arithmetic using unsigned doesn't always work. Are you sure the underlying code doesn't use a `-1` as a marker somewhere? If it has any "count down" for loops those won't work with unsigneds. 

Minor nit: we use `unsigned` rather than `unsigned int`.